src/app/components/ScrollablePromo/index.test.jsx

```diff
@@ -127,12 +199,14 @@ describe('ScrollablePromo', () => {
         <ScrollablePromoWithContext
           blocks={edOjA.model.blocks}
           blockGroupIndex={1}
+          service="news"
```

Comment:
I know this is test data so it doesn't matter as much but I thought service would be something like `hindi`, `pidgin` etc. `new` isn't a service, is it?

Reply:
Yes, news (English) is one of the 44 services we support, [here](https://confluence.dev.bbc.co.uk/display/NEWSWORLDSERVICE/A11y+Swarms ) is a list of all services.